microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -23,9 +23,11 @@ If you use the Evergreen runtime, before running your WebView2 app, test whether
 <!-- ====================================================================== -->
 ## Run compatibility tests regularly when using the Evergreen Runtime
 
-When using the Evergreen WebView2 Runtime, the runtime updates automatically, so you must regularly run compatibility tests.  To ensure that your WebView2 app will continue to work as expected, test your web content in the WebView2 control against [Microsoft Edge Insider (preview) Channels](https://www.microsoft.com/edge/download/insider) (Beta, Dev, or Canary).
+When using the Evergreen WebView2 Runtime, the runtime updates automatically, so you must regularly run compatibility tests.  To ensure that your WebView2 app will continue to work as expected, test your web content in the WebView2 control against the Microsoft Edge preview channels (Beta, Dev, or Canary).  The  preview channels are also called _Insider channels_.
 
-This guidance is similar to the guidance that we give to web developers.  See [Test your app for forward-compatibility](../concepts/distribution.md#test-your-app-for-forward-compatibility).
+The above guidance is similar to the guidance for web developers.  See [Test your app for forward-compatibility](../concepts/distribution.md#test-your-app-for-forward-compatibility).
```

Comment:
```suggestion
The above guidance is similar to the guidance for web developers.  For best practices on how to test your app for forward-compatibility, see [Prerelease testing using preview channels](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/prerelease-testing) and [Self-host by deploying preview channels](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/self-hosting).
```

The current link just links to a similar sounding section, that links to our best practices. So my suggestion is to skip the need to jump to two links, and jump straight to one:

![image](https://github.com/MicrosoftDocs/edge-developer/assets/5773562/3e1ed025-847b-4674-9b07-96fffed8dd1c)